.github/workflows/Pkg EdFi.Ods.Extensions.TPDM.yml

```diff
@@ -0,0 +1,175 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: Pkg EdFi.Ods.Extensions.TPDM
+
+on:
+  push:
+    branches:
+      - main
+  
+  workflow_dispatch:
+    inputs:
+      distinct_id:
+        description: 'distinct ID for Rebuild Database Templates workflow'
+        required: false
+        default: 'distinct_id'
+          
+        
+env:
+  INFORMATIONAL_VERSION: "7.1"
+  BUILD_INCREMENTER: "1"
+  CONFIGURATION: "Release"
+  HEAD_REF:  ${{ GITHUB.HEAD_REF }}
+  REF_NAME:  ${{ GITHUB.REF_NAME }}
+  REPOSITORY_DISPATCH_BRANCH: ${{ github.event.client_payload.branch }}
+  REPOSITORY_OWNER: ${{ GITHUB.REPOSITORY_OWNER }}
+
+jobs:
+  FindStandardAndExtensionVersions:
+    runs-on: ubuntu-latest    
+    outputs:
+      StandardVersions: ${{ steps.Set_StandardVersions.outputs.StandardVersions }}
+      ExtensionVersions: ${{ steps.Set_ExtensionVersions.outputs.ExtensionVersions }}
+    steps:
+    - name: Checkout Ed-Fi-Extensions
```

Comment:
This appears to be checking out the Ed-Fi-Alliance-OSS/Ed-Fi-Extension repo where we store TPDM-Core (maybe? I don't see a repository tag, so not sure what this doing). But if we are using this for determining version, it will be incorrect as TPDM-Core will be v1.1 and TPDM-Community will be v1.2

Reply:
This just gets sets the StandardVersions and ExtensionVersions variables from the previous job. 
StandardVersions is extracted using a PS script from the Ed-Fi-ODS repo but I've hardcoded it to '5.0.0' since we've no Artifacts for 4.0.0
ExtensionVersions is extracted by the build.githubactions.ps1 from the EdFi.Ods.Extensions.TPDM.csproj file. So, no. It's not pulling Core